### DISCUSSION TITLE
Host Migration

### DISCUSSION THREAD

**Discussion by engineer330426** [CrowdStrike/falconpy#1037] [Ideas]
I know on the PSFalcon side of the house there are srcipts for host migration(Move host from one cid to another) I reviewed the documentation from FalconPY, and I don't see anything similar to this.

The process flow is, as follows:
Check if host is online -> Does it have installer (If not download) -> Put Script to uninstall current sensor and install new sensor

End result is you have the newly installed sensor reporting to the proper CID and the old sensor no longer reporting to the old CID.  (Manually delete from old CID)

**Comment by jshcodes** [comment]
This is an excellent sample suggestion. Adding this to our punch list.

Thank you! 🙇 😄

**Comment by engineer330426** [comment]
Thank you Josh,  Always appreciate the hard work and effort you and your
team put forth.

On Fri, Sep 1, 2023 at 10:10 AM Joshua Hiller ***@***.***>
wrote:

> This is an excellent sample suggestion. Adding this to our punch list.
>
> Thank you! 🙇 😄
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CrowdStrike/falconpy/discussions/1037#discussioncomment-6886293>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBMAVUG6HWFHGALAOMNVRX3XYHUGJANCNFSM6AAAAAA4HN5GDU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Best Regards,

Ryan Reid
Sr. Security Systems Admin
(M) 443-262-5891
MD DoIT | DMI Contractor
